Clients: expose RSE limits in rucio-admin. Closes #4073

davidpob99 opened the pull request:
### Clients: expose RSE limits in rucio-admin. Closes #4073

When running `rucio-admin rse info RSE`, now it prints the quota for each account, both the local and global scopes.

Example: `rucio-admin rse info XRD1`
Output:
```
[...]
Account limits:
===============
  jdoe
    local_limit: 512.000 B
    global_limit: 0.0 B
  panda
    local_limit: 0.0 B
    global_limit: 1.024 kB
  root
    local_limit: Inf
    global_limit: 0.0 B
```

davidpob99 committed:
Clients: expose RSE limits in rucio-admin #4073

dchristidis reviewed (commented):
Welcome @davidpob99. This modification is unwanted and does not address the reported issue. Please re-read the description (including the necessity to add two new commands). Any questions should go into the issue itself instead of this PR.

davidpob99 committed:
Revert "Clients: expose RSE limits in rucio-admin #4073"

This reverts commit 30f66678f6feca82106716823256039511db3b5c.

davidpob99 committed:
Clients: expose RSE limits in rucio-admin. set-limit, delete-limit #4073

Created two new subcommands: set-limit and delete-limit and added to the report in rse info

davidpob99 commented:
Amend my misunderstanding in the latest commit. Now I created two new commands: `set-limit` and `delete-limit`:
- `rucio-admin rse set-limit <rse> <name> <value>`
- `rucio-admin rse delete-limit <rse> <name>`

Add to the `rse info` report the RSE limits:
```
RSE limits:
===========
  MaxBeingDeletedFiles: 1000
  MinFreeSpace: 10000
```

Modify the Flask API because the delete limit function ([lib/rucio/client/rseclient.py](https://github.com/rucio/rucio/blob/721b6dfa7e34853791531a6d6529e05964e988a3/lib/rucio/client/rseclient.py#L591)) for the RSE limits did not work:
```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>405 Method Not Allowed</title>
<h1>Method Not Allowed</h1>
<p>The method is not allowed for the requested URL.</p>
```
Add 'delete' string in [lib/rucio/web/rest/flaskapi/v1/rses.py](https://github.com/rucio/rucio/blob/721b6dfa7e34853791531a6d6529e05964e988a3/lib/rucio/web/rest/flaskapi/v1/rses.py#L878):
```python
bp.add_url_rule('/<rse>/limits', view_func=limits_view, methods=['get', 'put', 'delete'])
```
Add some tests.

bziemons commented:
@davidpob99 some tests seem to be failing & please check the syntax problems

davidpob99 committed:
Fix tests and syntax problems

davidpob99 committed:
Fix error in tests with sqlite
